third_party/nist_cavp_testvectors/BUILD

```diff
@@ -0,0 +1,3 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this empty BUILD file intentional?

Reply:
Yes, the BUILD file is needed for bazel to recognize this directory as a package. Otherwise, you get
```
ERROR: Error computing the main repository mapping: Label '//third_party/nist_cavp_testvectors:repos.bzl' is invalid because 'third_party/nist_cavp_testvectors' is not a package; perhaps you meant to put the colon here: '//:third_party/nist_cavp_testvectors/repos.bzl'?
```